spark-3/src/main/scala/org/neo4j/spark/DataSource.scala

```diff
@@ -8,11 +8,13 @@ import org.apache.spark.sql.types.StructType
 import org.apache.spark.sql.util.CaseInsensitiveStringMap
 import org.neo4j.spark.service.SchemaService
 import org.neo4j.spark.util.Validations.validateConnection
-import org.neo4j.spark.util.{DriverCache, Neo4jOptions}
+import org.neo4j.spark.util.{DriverCache, Neo4jOptions, Validations}
 
 class DataSource extends TableProvider
   with DataSourceRegister {
 
+  Validations.version(Seq("3.0", "3.1"))
```

Comment:
Why simply just `3`?
```suggestion
  Validations.version(Seq("3.1"))
```